### ISSUE TITLE
New maintainers wanted!

### ISSUE THREAD

**Issue by lunaryorn** [cask/cask#376]
Dear Emacs & Cask community,

TL;DR: @rejeep and me are stepping down as Cask maintainers and are looking for new maintainers.

Cask has become a core tool for Emacs package developers, yet development activity in this repository has ever decreased since quite some time.  @rejeep and me have less and less time to dedicate to Cask; we can't work on new features, and often struggle to even find time to review incoming pull request.  We have to admit that we are not longer able to maintain Cask at the level the Emacs community deserves. 

Consequently we need to step down as Cask maintainers, and are looking for motivated people to take over.  I will step down immediately (I'm not watching this repository anymore and will waive commit access soon), and @rejeep will continue to answer questions and try to review pull requests until new maintainers are found.

If you're interested please join this thread, and we'll make sure to smoothly transition Cask to new maintainers.

Inviting @purcell, @tarsius, @magnars, @bbatsov, @phillord, @syohex and @Fuco1 to make you aware, and to spread the information to the wider community.

Greetings,
@rejeep and @lunaryorn

**Comment by randre03** [comment]
Hello, I've used Cask before but am just starting with elisp (familiar with other lisps). I'd be happy to help out!

**Comment by joelmccracken** [comment]
I've hacked on Cask a bit. I think I may have submitted a patch a couple of years ago, I'm not sure. I'm not the most qualified person, but I think Cask is really important and would work to maintain it.

I guess what I'm saying is, I will volunteer as maintainer, if no one else more qualified steps forward!

**Comment by lunaryorn** [comment]
Thanks folks, that's amazing!

**Comment by bharath1097** [comment]
I use Cask a lot and i use python in my day job and i am also familiar with elisp. I'd be happy to help out.

**Comment by dgellow** [comment]
Maybe, add a note in the Readme to let people know the project is looking for a new maintainer?

**Comment by joelmccracken** [comment]
I recently realized that `use-package` and `cask` offer similar value. Its hard for me to be sure that `use-package`. Can anyone comment on the value of working on `cask` and not adopting `use-package`?

**Comment by Fuco1** [comment]
I don't quite see how the two relate.  `use-package` is a configuration macro while `cask` is a build tool.

How would you use `use-package` in a CI environment to install dependencies and set up load paths and run tests?

How would you use `cask` to configure a package, e.g. smartparens or flycheck?

**Comment by joelmccracken** [comment]
I should have explained this: look at the `:ensure` and `:pin` options.

**Comment by dgellow** [comment]
Oh, that's great. I didn't know those options. At least for my use case (user who wants to share his config between multiple machines) it should do the job

**Comment by phillord** [comment]
@joelmccracken I use both -- use-package for configuring my working Emacs, and cask as a build tool and execute tool for doing CI, packaging and so forth. I've never really used cask as a non-developer tool. As you say `:ensure` does that job.

**Comment by rejeep** [comment]
@joelmccracken Use-Package can be used instead of Cask for your personal configuration. I use both actually. The real important issue that Cask solves (that Use-Package) does not is a proper way of developing Emacs packages.

**Comment by sambrightman** [comment]
Closing this for now, since I've been doing maintenance.
